datarepo-client/build.gradle

```diff
@@ -118,3 +118,9 @@ clean {
     delete "pom.xml"
     delete "src/"
 }
+
+project(":datarepo-client") {
+    sonar {
+        skipProject = true
+    }
+}
```

Comment:
I'm surprised that this is needed. I would expect that if the `sonar` gradle plugin isn't listed above in `plugins` then it would need to be configured here. Maybe this is needed due to the way the TDR subprojects are set up?

Reply:
I thought that would be the case as well, but the sonar logs indicated it was trying to picking the client code. Adding this removed the logs.